lib/tasks/annual_fund_impact_metrics_activities.rake

```diff
@@ -0,0 +1,43 @@
+require "csv-safe"
+
+namespace :activities do
+  task :annual_fund_impact_metrics, [:output_csv_path] => :environment do |_task, args|
+    desc "Generates a CSV file of Activities that will be used to pre-populate an annual
+          collaborative spreadsheet of fund impact metrics"
+
+    csv_path = args[:output_csv_path] || "tmp/activities.csv"
```

Comment:
Is it worth giving the filename a more descriptive default? Such as `activities_for_impact_metrics.csv` for example.

Reply:
Yep, I really struggled with naming the task as well. What do you think of that `annual_fund_impact_metrics`?